json_serializable/lib/src/utils.dart

```diff
@@ -118,6 +118,8 @@ JsonSerializable mergeConfig(JsonSerializable config, ConstantReader reader) {
 bool isEnum(DartType targetType) =>
     targetType is InterfaceType && targetType.element.isEnum;
 
+bool isList(DartType targetType) => targetType.isDartCoreList;
```

Comment:
Delete this new helper. Just put this `isDartCoreList` call inline above.

Reply:
Done!